src/movableCoord.js

```diff
@@ -183,6 +183,10 @@ eg.module("movableCoord", ["jQuery", eg, window, "Hammer"], function($, ns, glob
 						this._subOptions = options;
 						this._status.curHammer = hammer;
 						this._panstart(e);
+					} else if (e.isFinal && e.direction !== MC.DIRECTION_NONE &&
+						!(e.direction & this._subOptions.direction)) {
+						// Although direction of movement is different from options.direction, MC should fire 'release' event.
+						this._panend(e);
```

Comment:
@jongmoon you'd better make `_triggerRelease` method like this
```js
_triggerRelease: function(e) {
this._setInterrupt(false);
this.trigger("release", {
	//...
});
```
and I find potential bug due to your PR.
we should also initalize `this._status.moveDistance`
thanks